app/DTOs/ApiResultsResponse.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\DTOs;
```

Comment:
Should be App\DTO, not App\DTOs -- everything else about the PR looks good now tho, so just need to fix this one.